packages/core/src/components/message-bar/message-bar.scss

```diff
@@ -21,7 +21,6 @@
     flex-wrap: nowrap;
     justify-content: space-between;
     min-height: 3.375rem;
-    margin: $small-space $small-space 0rem $small-space;
```

Comment:
This will cause a visual breaking change. Pull it up into host element before it can be removed finally in v3.